core/src/main/java/overflowdb/NodeDb.java

```diff
@@ -248,57 +236,60 @@ public void markAsClean() {
                             Edge edge,
                             int blockOffset,
                             String key) {
-    int propertyPosition = getEdgePropertyIndex(direction, edge.label(), key, blockOffset);
+    AdjacentNodes adjacentNodes = this.adjacentNodes;
```

Comment:
This does the right thing, but can we choose a different name for the temp? Just for readability -- make it obvious that we do the right thing, and make it easy to grep for accesses to the instance variable in order to hunt for violations.

Reply:
sure, no worries

Reply:
just renamed those ones to `adjacentNodesTmp`